akirachix/Tech-Marvels-Informational
# Feature/product

# Engineers Checklist

## What does this PR do?
- It shows our products: the mobile app and the web portal.

## How can it be tested?
- The mobile app is for farmers to use. It enables them to check their records and keep up to date with their transactions.
- The web portal is for cooperatives and Sacco. Helps cooperatives update the farmer's records and saccos retrieve the farmer's information.

# Where is it documented?
- Wezesha App

![Screenshot from 2024-07-02 16-01-54](https://github.com/akirachix/Tech-Marvels-Informational/assets/158448884/6dd22387-5b2e-4188-a5f9-50dec60ad42a)
